app/builders/immediate_email_builder.rb

```diff
@@ -20,6 +20,7 @@ def call
   def records
     recipients_and_content.map do |recipient_and_content|
       [
+        Email.statuses["pending"],
```

Comment:
I believe you can actually use the symbol `:pending` here.

Reply:
Unless this is for performance?

Reply:
oh really? as in just `:pending` or `Email.statuses[:pending]`? Be cool if I can do the former.

Reply:
The former, just `:pending`, I used that in the import script!

Reply:
Awesome, will try that out 👍